tests/pools/test_pool_rpc.py

```diff
@@ -775,7 +744,6 @@ async def status_is_self_pooling():
                 return pw_status.current.state == PoolSingletonState.SELF_POOLING.value
 
             await time_out_assert(timeout=WAIT_SECS, function=status_is_self_pooling)
-            pw_info: PoolWalletInfo = (await client.pw_status(wallet_id))[0]
```

Comment:
I get the impression `flake8` complained about the variable not being used. Do you think it would make sense to still make the function call and discard the return value?

Reply:
This is just an artifact of a lot of debugging when these tests were not working (during pool development). These calls are not needed and can be safely removed.